src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -19,29 +19,29 @@ const MetaTraderConfig = (() => {
             const standard_config = {
                 account_type: 'standard',
                 leverage    : 1000,
-                short_title : localize('Standard'),
+                short_title : localize('Financial'),
```

Comment:
are we changing this in deriv as well?

Reply:
@mustofa-binary yes, the card is in deriv for that

Reply:
deriv.com also need to change then, let me create a card for it as well

Reply:
@mustofa-binary thanks, but please add a don't merge label as we want to make sure everyone is onboard and aware before we merge any of them